packages/client/mutations/SignUpWithPasswordMutation.ts

```diff
@@ -45,10 +46,10 @@ const SignUpWithPasswordMutation: StandardMutation<
     onError,
     onCompleted: (res, errors) => {
       const {acceptTeamInvitation, signUpWithPassword} = res
-      const {error: uiError} = signUpWithPassword
+      const {error: uiError, user} = signUpWithPassword
       onCompleted({signUpWithPassword}, errors)
       handleAcceptTeamInvitationErrors(atmosphere, acceptTeamInvitation)
-      ReactGA.event('sign_up')
+      ReactGA.event('sign_up', {isPatient0: user?.isPatient0})
```

Comment:
+1 should this be moved inside the `if` below so we know the sign up was successful?